tests/testthat/test-GeospatialFunctions.R

```diff
@@ -0,0 +1,169 @@
+# Testing the Geospatial Functions ----
+# testing `GeoSpatialFunctions.R` with the same data that was used in the 
+# `TADAGeospatialFunctions.Rmd` Vignette in chunk 6 (3/18/24).
+
+# TADA_dataframe <- TADA_DataRetrieval(
+#   startDate = "2020-01-01",
+#   endDate = "2020-12-31",
+#   characteristicName = "pH",
+#   countycode = "US:08:069",
+#   applyautoclean = TRUE
+# )
+
+# Read in sample data for tests
+TADA_dataframe <- readRDS(testthat::test_path("testdata/GeospatialFunctions_TADA_dataframe.rds"))
+TADA_spatial <- readRDS(testthat::test_path("testdata/GeospatialFunctions_TADA_spatial.rds"))
+TADA_with_ATTAINS <- readRDS(testthat::test_path("testdata/GeospatialFunctions_TADA_with_ATTAINS.rds"))
+TADA_with_ATTAINS_list <- readRDS(testthat::test_path("testdata/GeospatialFunctions_TADA_with_ATTAINS_list.rds"))
+
+# TADA_MakeSpatial ----
+
+test_that(desc = "`TADA_MakeSpatial()` converts the TADA_dataframe into an {sf} object",
```

Comment:
MB pointed this out to me, but for TADA we will want to avoid {package} to remain consistent with TADA's current style.